qtglean/pings.yaml

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This file defines the pings that are recorded by the Glean SDK.
+# Their code APIs is automatically generated, at build time using,
```

Comment:
nitpick:
```suggestion
# Their code APIs is automatically generated at build time, using
```

Reply:
Thanks, good catch. (I did a copy paste from the other `pings.yml` file, and should have looked over the comments more carefully.)